src/lib/y2network/s390_device_activators/qeth.rb

```diff
@@ -49,8 +49,7 @@ def configure_attributes
         extra_attributes.concat(attributes.split(" ")) if attributes
         # Only set if enable
         extra_attributes << ipa_takeover_attribute if ipa_takeover
-        # Only set if enable
-        extra_attributes << layer2_attribute if layer2
+        extra_attributes << layer2_attribute
```

Comment:
I would add here comment with link to bugzilla that attribute always has to be set